packages/manager/modules/vrack/src/vrack-associated-services/ipv6/slaac-help.html

```diff
@@ -0,0 +1,12 @@
+<p data-translate="vrack__assigned-ipv6__subrange_slaac_help"></p>
+<a
+    class="oui-link_icon"
+    aria-hidden="true"
```

Comment:
according to sonar analysis, remove `aria-hidden` attribute and replace it with `rel` attribute
```suggestion
   rel="noopener"
```